nixos/modules/services/misc/forgejo.nix

```diff
@@ -621,14 +627,10 @@ in
       environment = {
         USER = cfg.user;
         HOME = cfg.stateDir;
-        # `GITEA_` prefix until https://codeberg.org/forgejo/forgejo/issues/497
-        # is resolved.
-        GITEA_WORK_DIR = cfg.stateDir;
-        GITEA_CUSTOM = cfg.customDir;
```

Comment:
We still need those.

The comment was referencing the `GITEA_` prefix instead of `FORGEJO_`.
You are right that https://codeberg.org/forgejo/forgejo/issues/497 has been resolved for quite some months now.

But this just means we can now use
```nix
        FORGEJO_WORK_DIR = cfg.stateDir;
        FORGEJO_CUSTOM = cfg.customDir;
```

Reply:
Fixed this!